internal/core/metadata/router.go

```diff
@@ -38,7 +39,7 @@ func LoadRestRoutes(dic *di.Container) *mux.Router {
 
 	// Configuration
 	r.HandleFunc(clients.ApiConfigRoute, func(w http.ResponseWriter, r *http.Request) {
-		configHandler(w, container.LoggingClientFrom(dic.Get))
+		configHandler(w, container.LoggingClientFrom(dic.Get), config.ConfigurationStruct{})
```

Comment:
Why are you passing an empty config.ConfigurationStruct{} here?

Commented here; occurs elsewhere.

Reply:
I am mortified by this oversight (No excuses here, but bulk refactoring, compounded by inadvertence, contributed to this oversight.) Rectified.